src/seismometer/configuration/model.py

```diff
@@ -171,6 +172,17 @@ class Event(BaseModel):
     The strategy for aggregating (or selecting) scores for an event.
     Supports min, max, first, and last; defaulting to max.
     """
+    merge_strategy: Optional[MergeStrategies] = "forward"
+    """
+    The strategy for merging events with predictions.
+    Supports first, last, nearest, forward, and count; defaulting to forward.
```

Comment:
I think you'll need a blank line after this for sphinx not to choke on the change in indentation

Reply:
Fixed in 49a9da1